ui/src/main/webapp/src/app/reports/ignoredFiles/ignoredFiles-report.component.html

```diff
@@ -0,0 +1,94 @@
+<!-- Will be replaced by shared styles -->
```

Comment:
Please remove obsolete styles. Shared styles are already in master.

Reply:
Done